miden/src/cli/verify.rs

```diff
@@ -62,4 +68,34 @@ impl VerifyCmd {
 
         Ok(())
     }
+
+    fn infer_defaults(&self) -> Result<(PathBuf,PathBuf),Report> {
+        let proof_file = if Path::new(&self.proof_file.as_os_str()).try_exists().is_err() {
+            return Err(Report::msg("Proof file does not exist"));
+        } else {
+            self.proof_file.clone()
+        };
+
+        let base_name = proof_file.file_stem().expect("Invalid proof file").to_str().unwrap();
+
+        let input_file = self.input_file.clone().unwrap_or_else(|| {
+            let mut input_path =
+                proof_file.parent().unwrap_or_else(|| Path::new(".")).to_path_buf();
+                input_path.push(base_name);
+                input_path.set_extension("inputs");
+            // input_path.push(format!("{}.inputs", base_name));
+            input_path
+        });
```

Comment:
Why do we need to first to get the base name and then modify it. Would something like this not work?:
```rust
let input_file = self.input_file.unwrap_or_else(|| {
    let mut input_path = proof_file.clone();
    input_path.set_extension("inputs");
    input_path
});
```

Reply:
You are right. This makes the code more compact.
Have made the necessary changes